README.md

```diff
@@ -18,6 +18,12 @@ pip install --user pydockenv
 
 To avoid conflicts this installs `pydockenv` to the Python user install directory. In order to run the `pydockenv` binary, you will need to have that directory in your `PATH`.
 
+`pydockenv` support python 3 at the moment so in case your system have python 2 installed, the binary `python` will be linked
+to python 2 so to run it with a different interpreter you do it exporting `PYDOCKENV_INTERPRETER` like:
+```
+export PYDOCKENV_INTERPRETER=path/to/binary
+```
+To have it permanently write it in you shell configuration file.
```

Comment:
I'd delete this as it's probably not what the user wants to do.
```suggestion
```

Reply:
done